Assets/BoundingsTimeout2D.cs

```diff
@@ -8,7 +8,6 @@ public class BoundingsTimeout2D : MonoBehaviour
 
     void Start()
     {
-        timeout = 3;
```

Comment:
Making changes to variables in start will override the editor settings when the object is initialized. I am not quite sure where the unity magic lies, but I think the editor variables are being assigned to the variable declaration...
```
public int timeout;
public int timeout = <editor-value>;
```
This gets overwritten on Start().
I did a bit of googling on setting editor defaults last night but couldn't find anything, though int's do seem to default to 0.